src/instances/RaidInstance.ts

```diff
@@ -1987,6 +1987,13 @@ export class RaidInstance {
 
             this._pplConfirmingReaction.add(i.user.id);
             const res = await confirmReaction(i, this._allEssentialOptions, this._modifiersToUse);
+            if(!this._raidVc){
+                await i.editReply({
+                    content: "The raid you are attempting to react to has been closed or aborted.",
+                    components: []
+                });
+                return;
+            };
```

Comment:
Minor styling issue -- semicolon not needed for end brace like this. Doesn't matter too much, we'll deal with this later.

Reply:
Got it.